pcdsdevices/interface.py

```diff
@@ -241,6 +243,17 @@ def get_value(signal):
     return None
 
 
+def get_units(signal):
+    attrs = ('units', 'egu', 'derived_units')
```

Comment:
I never realized we had so many places to store this info

Reply:
I _think_ for the most part it'll get it right on the first `metadata['units']` attempt, but to be honest I just shrugged and thought _meh, assume nothing, try all the places_.